src/main/webapp/step3.html

```diff
@@ -0,0 +1,55 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Home</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <nav>
+
```

Comment:
Empty line? Also empty nav?

Reply:
The nav will be added later and it will be common for all the pages. Let me add a TODO! :)

Reply:
Resolved!
TODO: reference the new PR as soon as it's done

Reply:
#14